tests/console/yast2_kdump_accept_kdump_options.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Nothing change and accept kdump options.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+use cfg_files_utils qw(validate_cfg_file);
+use scheduler qw(get_test_suite_data);
+use testapi qw(save_screenshot select_console);
+use utils;
+use YaST::Module;
+
+sub run {
+    my $kdump = $testapi::distri->get_kdump();
+
+    YaST::Module::open(module => 'kdump', ui => 'ncurses');
+
+    $kdump->accept();
+    save_screenshot;
+
+    select_console('root-console');
+    # check service (without restarting)
```

Comment:
why we need this comment everywhere? if restarted would be need it, we would code it... I would remove it or give a better explanation.